README.md

```diff
@@ -654,6 +654,74 @@ instance.delete()
 
 **Do not use delete() in production!**
 
+Bulk-updating sandboxes
+-----------------------
+
+OpenCraft IM provides a tagging mechanism that you can use to update instances in bulk.
+The following script shows how to update multiple instances at once from the Django shell:
+
+```python
+# Author: Brandon DeRosier <bdero@opencraft.com>
```

Comment:
@antoviaque 

> If there are things that are different each time, can we cleanly separate them from the workflow of rebuilding all the instances?

Somewhat. For example, in this case, we could separate the tagging of instances and the changing of settings (steps 1, 2, and 8) from the process of spawning and activating the new appservers (steps 3-7). So steps 3-7 could be put into a management command (or, better, an API endpoint which is fairly easy to do and also gives us [an autogenerated UI](https://console.opencraft.com/api/v1/)).

There are cases where that approach won't work, though: for example, if an upgrade includes a migration that's not backwards compatible. Then the script needs to deploy a new instance without migrations, take the current instance offline (display a maintenance page), run the migrations on the new instance, and point the LB to the new instance, which is a completely different process than we're seeing here. So even if we put steps 3-7 into a command/API, there is a need to document how to do these things via a flexible script.